namer/core/src/main/scala/io/buoyant/hostport.scala

```diff
@@ -0,0 +1,73 @@
+package io.buoyant
+
+import com.twitter.finagle.{Name, NameTree, Namer, Path}
+import com.twitter.util.{Activity, Try}
+
+/**
+ * A helper
+ */
+trait RewritingNamer extends Namer {
```

Comment:
ah, I see.  Pretty confusing that it's in hostport.scala though.  How about giving it its own file?